<?xml version="1.0"?>
<launch>

	<!-- send parameters to parameter server -->
	<rosparam command="load" ns="sdh_controller" file="$(find cob_sdh)/ros/launch/$(env ROBOT).yaml"/>

	<!-- start sdh node -->
	<node pkg="cob_sdh" type="cob_sdh" name="sdh_node" ns="sdh_controller" cwd="node" respawn="false" output="screen" />

	<!-- rostest integration -->
	<include file="$(find cob_sdh)/ros/launch/sdh.test" />

</launch>
